### ISSUE TITLE
Sort and Resize buttons are not accessible using the keyboard keys: A11y_Aspire Dashboard_Resources_Keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/project-system#9694]
### GitHubTags
#A11ySev1; #A11yTCS;#DesktopWeb; #Win32;#.NET Project system for CSharp and VB;#BM_AspireDashboard_Web_June25;#Win32;#WCAG2.1.1;#Keyboard;#A11yMAS;

### Environment Details:
Application Name: Aspire Dashboard
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.4351

### Repro Steps:

1. Open Visual studio.
2. Open a aspire project and run it. 
3. A new window will open in the chromium edge, TAB to "Resources" tab item and hit ENTER key.
4. Verify that Sort and Resize buttons are not accessible using the keyboard keys.

### Actual Result:
Sort and Resize buttons are not accessible using the keyboard keys

### Expected Result:
Sort and Resize buttons should be accessible using the keyboard keys.

### User Impact:
Keyboard and screen reader users will not be able to access the controls.

### Attachment:

**Comment by adamint** [comment]
@AnKushSingh05 please file issues in dotnet/aspire
